<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAP_GUI_EntityInfoView</class>
 <widget class="QWidget" name="MAP_GUI_EntityInfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>18</number>
   </property>
   <property name="verticalSpacing">
    <number>7</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="planeEntity">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="3" column="1">
       <widget class="QLineEdit" name="aveSpeedLineEditP"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="maxSpeedLineEditP"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="insSpeedLineEditP"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="maxWeightLineEditP"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="radiusLineEditP"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="radiusLabelP">
        <property name="text">
         <string>作战半径：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="maxSpeedLabelP">
        <property name="text">
         <string>最大速度：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="aveSpeedLabelP">
        <property name="text">
         <string>平时爬行速度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="classLabelP">
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="maxWeightLabelP">
        <property name="text">
         <string>最大重量：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabelP">
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="insSpeedLabelP">
        <property name="text">
         <string>瞬时爬行速度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="classLineEditP"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEditP"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IDLabelP">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDLineEditP">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="shipEntity">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="2" column="1">
       <widget class="QLineEdit" name="classLineEditS"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="radiusLineEditS"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabelS">
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="maxSpeedLabelS">
        <property name="text">
         <string>最大航速</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="classLabelS">
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEditS"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="maxWeightLineEditS"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="radiusLabelS">
        <property name="text">
         <string>作战半径：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="maxSpeedLineEditS"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="maxWeightLabelS">
        <property name="text">
         <string>最大重量：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IDLabelS">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDLineEditS">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="radarEntity">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="3" column="1">
       <widget class="QLineEdit" name="scopeSizeLineEditR"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="radiusLabelR">
        <property name="text">
         <string>探测半径：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="classLabelR">
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabelR">
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="scopeSizeLabelR">
        <property name="text">
         <string>可视探测范围（size）：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEditR"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="maxWeightLabelR">
        <property name="text">
         <string>最大重量：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="maxWeightLineEditR"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="scopeRearLineEditR"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="scopeRearLabelR">
        <property name="text">
         <string>可视探测范围（Rear）：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="radiusLineEditR"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="classLineEditR"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IDLabelR">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDLineEditR">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="groundEntity">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="5" column="0">
       <widget class="QLabel" name="heightLabelG">
        <property name="text">
         <string>高度：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEditG"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabelG">
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="widthLabelG">
        <property name="text">
         <string>宽度：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="communicationLabelG">
        <property name="text">
         <string>通信模式：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="communicationLineEditG"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lengthLineEditG"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="heightLineEditG"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="widthLineEditG"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="classLineEditG"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="classLabelG">
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="distanceLabelG">
        <property name="text">
         <string>距离：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lengthLabelG">
        <property name="text">
         <string>长度：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="distanceLineEditG"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IDLabelG">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDLineEditG">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>84</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
